src/atomic/main.cpp

```diff
@@ -455,6 +464,14 @@ int main(int argc, char **argv) {
   if(Zl!=0 || Zr !=0)
     printf("Done in %.6f\n",tnuc.get());
 
+  // Confinement potential
+  arma::mat Vconf(basis.Nbf(),basis.Nbf(),arma::fill::zeros);
+  if(conf_N) {
+    printf("Computing confinement potential\n");
+    arma::mat Vconf=basis.confinement(conf_N, conf_R, iconf, shift_pot);
```

Comment:
You are using a new variable here which screens the old one
```suggestion
    Vconf=basis.confinement(conf_N, conf_R, iconf, shift_pot);
```